plutus-chain-index-core/src/Plutus/ChainIndex/Api.hs

```diff
@@ -116,7 +117,7 @@ deriving instance OpenApi.ToSchema a => OpenApi.ToSchema (Page a)
 -- }
 data UtxoAtAddressRequest = UtxoAtAddressRequest
     { pageQuery  :: Maybe (PageQuery TxOutRef)
-    , credential :: Credential
+    , credential :: CardanoAddress
```

Comment:
```suggestion
    , address :: CardanoAddress
```

Reply:
It's a string: the serialised Bech32 of the address (defined in Cardano.Api.Address). The change for address aimed at getting rid of the partial conversion from Credential to address